content/docs/started/getting-started-azure.md

```diff
@@ -0,0 +1,7 @@
++++ title = "AWS For Kubeflow" 
```

Comment:
While you're here, please would you add the relevant bullet point to this page too:
https://www.kubeflow.org/docs/started/getting-started/

Reply:
New pull requested :)